addons/pharma/functions/fnc_inspectBreath.sqf

```diff
@@ -19,16 +19,15 @@
 params ["_medic", "_patient"];
 
```

Comment:
Look at Discord. There is a logical mistake in this file which needs to be updated before merging